src/gocart_seas_ngac_mod.F90

```diff
@@ -121,18 +126,12 @@ subroutine gocart_seas_ngac(ktau,dt,u_phy,              &
       end do
 
       ! -- add sea salt emission increments to existing airborne concentrations
-      chem_arr(p_seas_1) = chem_arr(p_seas_1) + tc(1)*converi
-      chem_arr(p_seas_2) = chem_arr(p_seas_2) + tc(2)*converi
-      chem_arr(p_seas_3) = chem_arr(p_seas_3) + tc(3)*converi
-      chem_arr(p_seas_4) = chem_arr(p_seas_4) + tc(4)*converi
-      chem_arr(p_seas_5) = chem_arr(p_seas_5) + tc(5)*converi
-
-      ! for output diagnostics kg/m2/s
-      emis_seas(p_eseas1) = bems(1)
-      emis_seas(p_eseas2) = bems(2)
-      emis_seas(p_eseas3) = bems(3)
-      emis_seas(p_eseas4) = bems(4)
-      emis_seas(p_eseas5) = bems(5)
+      do n = 0, number_ss_bins -1 
```

Comment:
Seems like this assumes that `number_ss_bins` and `num_emis_seas` are the same. So do we need both?

Reply:
I don't think so